data_utils/dataset.py

```diff
@@ -141,7 +152,14 @@ def merge_data(data, start, jump):
         tmp = data[data['Timestamp'] >= start].reset_index(drop=True)
         tmp = tmp[tmp['Timestamp'] < start + jump].reset_index(drop=True)
         if len(tmp) == 0:
-            return None, None, None, None, None
+            return None #, None, None, None, None
+        if len(tmp) == 1:
+            return tmp.iloc[0]
+            # _, high, low, open, close, volume = DataConverter.get_row_values(tmp.iloc[0])
+            # return high, low, open, close, volume
+
+        print("\033[91mMore than one row\033[0m")
```

Comment:
maybe this should throw error, as the following will no longer work if custom features are used.